test/lambdaisland/deep_diff/printer_test.clj

```diff
@@ -0,0 +1,21 @@
+(ns lambdaisland.deep-diff.printer-test
+  (:require [clojure.test :refer :all]
+            [lambdaisland.deep-diff.diff :as diff]
+            [lambdaisland.deep-diff.printer :as printer]))
+
+(defn- printed
+  [diff]
+  (let [printer (printer/puget-printer {})]
+    (with-out-str (-> diff
+                      (printer/format-doc printer)
+                      (printer/print-doc printer)))))
+
+(deftest print-doc-test
+  (testing "date"
+    (is (= "[31m-#inst \"2019-04-09T14:57:46.128-00:00\"[0m [32m+#inst \"2019-04-10T14:57:46.128-00:00\"[0m\n"
```

Comment:
I would use `\u001b` here instead of a literal escape character, makes it a bit easier to edit.